src/docs/release-notes/R26/FullNotes.mdx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+## GitHub Enterprise Migration
+
+-   Migrated our code base to a [new location](https://github.com/etn-ccis?q=blui), organized using GitHub Enterprise.
+
+## Documentation Site
+
+-   Added style guidelines for designing [app store logos](/style/logo).
+-   Added design guidelines for the [buttons](/patterns/buttons) design pattern.
+-   Added guidelines for how to request, conduct and interpret a [UI design audit](/design/design-audit).
+-   Added a page featuring a list of currently published [Brightlayer UI based projects](/community/blui-projects).
+
+## Figma Component Sticker Sheet
+
+-   Added new text field-based components, such as Physical Location Address Text Fields, Phone Number Text Fields, to the Figma component sticker sheet.
+-   Added a new component, App Logo Template, to help designers design their product app store logos.
+-   Added iPhone's "Dynamic Island" as a component.
+-   Added a Breadcrumb component.
+
+## Angular
+
+-   Updated our packages to support Angular 14.
+
+## React
```

Comment:
This was a bugfix, I would either leave it out or lump it under the bugs heading. It's a very low level fix to be highlighted in release notes.